mysql-test/suite/innodb/t/xtradb_compressed_columns_upgrade-master.opt

```diff
@@ -0,0 +1 @@
+--loose-compressed-columns-unknown1
```

Comment:
It looks like I re-invented the wheel. `--force-restart` does exactly what is needed here. I just didn't know about its existence.

`--loose-compressed-columns-unknown1` and `--loose-compressed-columns-unknown2` in the option files changed to `--force-restart`